logging/logging.go

```diff
@@ -420,6 +441,19 @@ func (instance *Logging) sendErrorResponse(errorStr, logID string) {
 	instance.logChannel <- response
 }
 
+func (instance *Logging) sendEmptyResponse(logID string) {
+	var part uint64 = 1
+
+	instance.logChannel <- cloudprotocol.PushLog{
+		LogID:      logID,
+		PartsCount: part,
+		Part:       part,
+		Content:    []byte{},
+	}
+
+	log.Debugf("Send empty log")
```

Comment:
Put log at the beginning of function. It is good practice in case the function crashes then it would be easier to identify the faulty function.